invenio_indexer/api.py

```diff
@@ -163,8 +163,11 @@ def bulk_delete(self, record_id_iterator):
         """
         self._bulk_op(record_id_iterator, 'delete')
 
-    def process_bulk_queue(self):
-        """Process bulk indexing queue."""
+    def process_bulk_queue(self, **kwargs):
+        """Process bulk indexing queue.
+
+        :param dict kwargs: Passed to ``elasticsearch.helpers.bulk``.
```

Comment:
Maybe worth an `intersphinx` like for [`kombu`](https://github.com/inveniosoftware/invenio-indexer/blob/master/docs/conf.py#L327)? Also I think is good to pass kwargs so we don't to change it later if we have to add more keywords to the ES bulk helper, but IMHO I would be explicit with the keyword and describe it in the `docstring`, what it does and that it goes to `elasticseacrh.helpers.bulk`!